app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.tsx

```diff
@@ -29,9 +30,12 @@ export const addConnectedState = (servers: JenkinsServer[]): JenkinsServer[] =>
 				if (originalServer) {
 					originalConnection = originalServer.name;
 				}
-			} else if (server.pluginConfig || server.pipelines.length) {
+			} else if (server.pluginConfig) {
 				connectedState = ConnectedState.CONNECTED;
 				if (ipAddress) ipAddressSet.add(ipAddress);
+			} else if (!server.pluginConfig && server.pipelines.length) {
+				connectedState = ConnectedState.UPDATE_AVAILABLE;
+				if (ipAddress) ipAddressSet.add(ipAddress);
```

Comment:
ipAddress shouldnt exist if server.pluginConfig is missing, so you could just strip this line oot